packages/client/src/components/ToolItem/ToolItem.component.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ToolItem.style.css';
+import { Link } from 'react-router-dom';
+
+export const ToolItem = ({
+  title,
+  timeFrameMin,
+  timeFrameMax,
+  groupSizeMax,
+  groupSizeMin,
+  pitch,
+}) => {
+  let firstCategoryClassName = 'not-selected-category';
+  let secondCategoryClassName = 'not-selected-category';
+  let thirdCategoryClassName = 'not-selected-category';
+  let fourthCategoryClassName = 'not-selected-category';
```

Comment:
There is quite a lot of repetitive code here and the category handling could be simplified.
If you intend to have the categories array here, something like this could be done:
```
{categories.map((category) => {
              return (
                <li>{`${category}`}</li>
              );
            })}
```

Then, either passing the Tool category to ToolItem as a prop or fetching it here from the DB, you could map the styling like this:
```
{categories.map((category) => {
let categoryClassName = ...
              return (
                <li className={`${categoryClassName}`}>{`${category}`}</li>
              );
            })}
```

Reply:
Handling it similarly to this ☝🏽 , code in 14-17 and 20-32 could be removed.

Reply:
New task added to JIRA for this

Reply:
https://hackyourfuture-dk.atlassian.net/browse/FP21-55